autoload/neomake/makers/ft/python.vim

```diff
@@ -278,8 +278,14 @@ endfunction
 " --fast-parser: adds experimental support for async/await syntax
 " --silent-imports: replaced by --ignore-missing-imports
 function! neomake#makers#ft#python#mypy() abort
+    let l:args = ['--check-untyped-defs', '--ignore-missing-imports']
+    let l:output = system('pyenv version')
```

Comment:
Should get cached.

Reply:
OTOH caching should get skipped here maybe, since you might be switching virtualenvs / $PATH?!